raft/src/main/java/org/apache/kafka/snapshot/FileRawSnapshotWriter.java

```diff
@@ -78,7 +85,11 @@ public void append(MemoryRecords records) {
             checkIfFrozen("Append");
             Utils.writeFully(channel, records.buffer());
         } catch (IOException e) {
-            throw new RuntimeException(e);
+            throw new UncheckedIOException(
+                String.format("Error writing file snapshot," +
```

Comment:
nit: missing a space after the comma. Also, it would be nice to be consistent with the use of comma vs period for the similar cases in this file

Reply:
Fixed!